server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/PlanNodeUtil.java

```diff
@@ -72,6 +72,9 @@ private static void nodeToString(PlanNode root, PrintContext ctx, StringBuilder
     }
     result.append(root.toString());
     result.append(System.lineSeparator());
+    if (root.getChildren() == null) {
```

Comment:
Seems better to use `for(int i = 0;i < root.getChildrenCount();i++)`

Reply:
yes. will fix it next pr